00_prepare_for_PyPI.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# This script must be run before packaging (python setup.py sdist upload).
+# This script must be run before packaging (twine upload dist/*).
```

Comment:
Well spotted!

This instruction generally fails, though: older versions are typically present in `dist/`, and `twine` PyPI rejects them. I'll update this line.